src/Fixer/PhpUnit/PhpUnitDedicateAssertInternalTypeFixer.php

```diff
@@ -93,22 +90,6 @@ public function getPriority()
         return -16;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function createConfigurationDefinition()
-    {
-        // @todo 3.0 drop `ConfigurableFixerInterface`
-        return new FixerConfigurationResolver([
-            (new FixerOptionBuilder('target', 'Target version of PHPUnit.'))
-                ->setAllowedTypes(['string'])
-                ->setAllowedValues([PhpUnitTargetVersion::VERSION_7_5, PhpUnitTargetVersion::VERSION_NEWEST])
-                ->setDefault(PhpUnitTargetVersion::VERSION_NEWEST)
```

Comment:
I would recover it.
it is a way that the default config is `VERSION_NEWEST`, so we can add support for future PHPUnit versions to the rule WITHOUT a need to change default config, which would be a BC breaker

Reply:
can you maybe prep a PR? because I broke my head around it for some time but while I agree with your spec I don't get the implementation

Reply:
yes
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5366
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5367

Reply:
thanks! :)